pkg/kube/multicluster/secretcontroller.go

```diff
@@ -196,7 +199,9 @@ func (c *Controller) processItem(key types.NamespacedName) error {
 	}
 	if exists {
 		log.Debugf("secret %s exists in informer cache, processing it", key)
-		c.addSecret(key, obj.(*corev1.Secret))
+		if err := c.addSecret(key, obj.(*corev1.Secret)); err != nil {
+			return fmt.Errorf("error adding secret %s error: %v", key, err)
```

Comment:
```suggestion
			return fmt.Errorf("error adding secret %s: %v", key, err)
```

nit: go error common practice  is to just add `: err`

Reply:
Thx, will fix  :-)